editor/utils/tracking.js

```diff
@@ -0,0 +1,61 @@
+/* eslint-disable no-console */
```

Comment:
I was thinking, just moving this to a root folder in gutenberg for now. And once we have a WP.org infrastructure we could move to the packages repository